nfs-ganesha will fail pkgconf test pipeline

If the pkgconf test pipeline is added to nfs it will fail in the following way:

```
2024/11/12 23:13:55 INFO running step "test/pkgconf"
2024/11/12 23:13:55 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:13:55 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:13:55 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:13:55 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:13:55 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + basename /home/build/melange-out/nfs-ganesha uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + dev_pkg=nfs-ganesha uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + apk info -L nfs-ganesha uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + grep -q ^Name: usr/lib64/pkgconfig/libntirpc.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + basename usr/lib64/pkgconfig/libntirpc.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + lib_name=libntirpc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + echo usr/lib64/pkgconfig/libntirpc.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 WARN + grep -q '^usr/lib/pkgconfig/libntirpc.pc$\|^usr/share/pkgconfig/libntirpc.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:13:55 INFO pod bb32c1566c8757afbcce76ec9e7d09e05132dd85864e15e36c16cbf27b2f64d8 terminated
2024/11/12 23:13:55 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:13:55 INFO   workspace dir: /home/user/tmp/melange-workspace-2447249271
2024/11/12 23:13:55 INFO   guest dir: /home/user/tmp/melange-guest-2465150400
2024/11/12 23:13:55 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/nfs-ganesha] Error 1
```

--- a/nfs-ganesha.yaml
+++ b/nfs-ganesha.yaml
@@ -1,7 +1,7 @@
 package:
   name: nfs-ganesha
   version: "6.5"
-  epoch: 0
+  epoch: 1
   description: NFS-Ganesha is an NFSv3,v4,v4.1 fileserver that runs in user mode on most UNIX/Linux systems
   copyright:
     - license: GPL-3.0-or-later
@@ -44,6 +44,9 @@ pipeline:
   - working-directory: src
     pipeline:
       - uses: cmake/configure
+        with:
+          opts: >
+            -DLIB_INSTALL_DIR="/usr/lib"
       - uses: cmake/build
       - uses: cmake/install
       - uses: strip
@@ -52,6 +55,9 @@ subpackages:
   - name: nfs-ganesha-dev
     pipeline:
       - uses: split/dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

--- a/nfs-ganesha.yaml
+++ b/nfs-ganesha.yaml
@@ -47,6 +47,7 @@ pipeline:
         with:
           opts: >
             -DLIB_INSTALL_DIR="/usr/lib"
+
       - uses: cmake/build
       - uses: cmake/install
       - uses: strip

